lib/route/domain.go

```diff
@@ -80,3 +80,34 @@ func (d *Domain) URL() url.URL {
 	}
 	return url.URL{Scheme: "http", Host: d.String()}
 }
+
+// Match returns true if the text satisfies the pattern
+// string, supporting only '*' wildcard in the pattern.
+func Match(pattern, name string) bool {
+	if pattern == "" {
+		return name == pattern
+	}
```

Comment:
🤔 i'm trying to think of a need for this. it may be a backwards compatible ... bug? i'd be happy returning false for empty patterns.

Reply:
That’s a fair point. Updated. I also took the time to un-export this function.